atlasdb-cassandra/src/main/java/com/palantir/atlasdb/cassandra/CassandraReloadableKvsConfig.java

```diff
@@ -39,7 +39,10 @@ public CassandraReloadableKvsConfig(
 
     @Override
     public CassandraServersConfigs.CassandraServersConfig servers() {
-        return config.servers();
+        // get servers from install config (for backcompat), otherwise get servers from runtime config
+        return config.servers().numberOfThriftHosts() > 0
+                ? config.servers()
+                : chooseConfig(CassandraKeyValueServiceRuntimeConfig::servers, null);
```

Comment:
nit: Can we just match other uses of `chooseConfig` to make this less confusing?
```suggestion
        if (config.servers().numberOfThriftHosts() > 0) {
            return config.servers();
        }
        return chooseConfig(CassandraKeyValueServiceRuntimeConfig::servers, config.servers());
```

Reply:
no - the other uses of `chooseConfig` prefer runtime over install

Reply:
Right, but we know `config.servers()` is empty at this point